<launch>
    <node name="level_planner" pkg="tel_first_half" type="level_planner_node" output="screen"/>
    <param name="level_code" type="int" value="2" />

    <node name="motorCtrl_node" pkg="wheel_tokyo_weili" type="motorCtrl_node" output="screen"/>
    <node name="encoderCounter_node" pkg="wheel_tokyo_weili" type="encoderCounter_node" output="screen"/>
    <node name="wheel_planner_node" pkg="wheel_tokyo_weili" type="wheel_planner_node" output="screen"/>
</launch>